python/README.md

```diff
@@ -85,6 +93,23 @@ def handler(event, context)
   ...
 ```
 
+**Fetching a pre-configured tracer anywhere**
+
+```python
+# handler.py
+from aws_lambda_powertools.tracing import Tracer
+tracer = Tracer(service="payment")
+
+@tracer.capture_lambda_handler
+def handler(event, context)
+  charge_id = event.get('charge_id')
+  payment = collect_payment(charge_id)
+  ...
+
+# another_file.py
+from aws_lambda_powertools.tracing import Tracer
+tracer = Tracer.instance()
```

Comment:
What is the behaviour if `another_file.py` is imported first? Would this fail?

I am thinking about the case where the Tracer is first instanciated in an import file, then re-used using `Trace.instance()` in another one. There shouldn't be a reason to enforce a specific import order.

Reply:
That's exactly why I made `Tracer.instance()` to create an instance if there isn't one so it wouldn't impact ordering :)

There's a **if** though. If you call `.instance()` first, it'll create a Tracer with default values unless there are env vars present. If you then create another `Tracer(explicit_config)` it'll create a new instance with a different config unless they're the same as env vars.

Reply:
As per our chat, moved to shared config instead (Borg)